contracts/RoT.sol

```diff
@@ -3,19 +3,19 @@ import "./Types.sol";
 
 contract RoT is Ownable {
     address public ESOPAddress;
-    address public addressOfCEO;
+    event ESOPAndCEOSet(address ESOPAddress, address ceoAddress);
 
-    // change esop contract
     function setESOP(address ESOP, address ceo) public onlyOwner {
-      addressOfCEO = ceo;
+      // owner sets ESOP and ceo only once then passes ownership to ceo
+      // initially owner is a developer/admin
```

Comment:
we do not have ESOP address when RoT is created, if this is what you mean. take a look at ESOP constructor. 
in the deployment process admin deploys both ESOP and RoT and passes ownership to CEO. I think I should keep CEO as separate role and do not transfer the ownership. Let me think.